core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -641,6 +646,33 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }
   }
 
+  def requestAccessToManagedGroup(groupRef: ManagedGroupRef): Future[PerRequestMessage] = {
+    dataSource.inTransaction { dataAccess =>
+      dataAccess.managedGroupQuery.load(groupRef).flatMap {
+        case None => {
+          throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.NotFound, s"The admins of the group ${groupRef.membersGroupName.value} were not found"))
```

Comment:
isn't is that the group is not found? Where do we validate that there are not special access instructions, i.e. calling this api is not valid for this group?